test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+--NAME--
+CM forwards PSB OK
+
+--SHELL-INIT--
+dbInit CB
+dbInit CM
+
+brokerStart CB 0-255 -multiservice
+brokerStart CM 0-255 -multiservice
+
+--SHELL--
+
+#
+# 01. Initial registration in CM, service-path /cm/cb
+# 02. Check the registration in CM, service-path /cm/cb
+# 03. Check the registration in CB, service-path /cm/cb
+# 04. Verify service-path in CB, test with erroneous service-path /cm/cb2
+# 05. Update registration in CM, service-path /cm/cb
+# 06. Check the registration in CM, service-path /cm/cb
+# 07. Check the registration in CB, service-path /cm/cb
+# 08. Verify service-path in CB, test with erroneous service-path /cm/cb2
+# 09. Check that everything is ok in database layer
+#
+
+
+echo "01. Initial registration in CM, service-path /cm/cb"
+echo "==================================================="
+payload='<?xml version="1.0" encoding="UTF-8"?>
+<registerContextRequest>
```

Comment:
Actually, it is a copy of an old test, made in XML.
I could translate it to JSON, I just don't see the need.
New tests in JSON, yes of course, but that is not really the case here.

NTC